go.mod

```diff
@@ -7,25 +7,15 @@ require (
 	bitbucket.org/creachadair/shell v0.0.6
 	github.com/Masterminds/semver/v3 v3.1.1
 	github.com/asaskevich/govalidator v0.0.0-20210307081110-f21760c49a8d
-	github.com/containerd/containerd v1.4.4
-	github.com/deislabs/oras v0.11.1
-	github.com/envoyproxy/go-control-plane v0.9.9-0.20201210154907-fd9021fe5dad
-	github.com/manifoldco/promptui v0.8.0
-	github.com/mattn/go-isatty v0.0.12
 	github.com/mattn/go-shellwords v1.0.11
 	github.com/mholt/archiver/v3 v3.5.0
 	github.com/mitchellh/go-homedir v1.1.0
-	github.com/opencontainers/image-spec v1.0.1
-	github.com/otiai10/copy v1.5.1
+	github.com/pkg/errors v0.9.1 // indirect
```

Comment:
wondering why pkg/errors comes up again?

Reply:
this is our logging library and we can remove it in another PR as it isn't needed to use an abstraction when you can't even configure it.